config/cmake/modules/MfemCmakeUtilities.cmake

```diff
@@ -138,6 +137,8 @@ macro(add_mfem_miniapp MFEM_EXE_NAME)
   # Actually add the executable
   mfem_add_executable(${MFEM_EXE_NAME} ${MAIN_LIST}
       ${EXTRA_SOURCES_LIST} ${EXTRA_HEADERS_LIST})
+  install(TARGETS ${MFEM_EXE_NAME}
+          RUNTIME DESTINATION miniapps)
```

Comment:
This should be done only if `MFEM_ENABLE_MINIAPPS` is true. I'm not sure if using `add_subdirectory(miniapps EXCLUDE_FROM_ALL)` excludes also the `install` commands or just the targets.

Reply:
It is, from the manual:

"If the EXCLUDE_FROM_ALL argument is provided, it has the following effects: [...]
Any install rules defined in the subdirectory or below will be ignored when installing the parent directory."